charmcraft/providers.py

```diff
@@ -224,14 +219,13 @@ def ensure_provider_is_available(provider: "Provider") -> None:
     :raises ProviderError: if provider is not available, or if the user
     chooses not to install the provider.
     """
+    # TODO: add provider.name and provider.install_recommendations to craft-providers
```

Comment:
@facundobatista - I had to reduce the helpfulness of these help messages and make a generic help message to get to the finish line.
Next week let's add your improvements to `craft-providers` and make these messages more helpful again.

Reply:
I created https://github.com/canonical/craft-providers/issues/166